### ISSUE TITLE
Cannot switch back to 1920x1080 native display on external monitors

### ISSUE THREAD

**Issue by deadlydog** [GeekyEggo/windows-display-settings#26]
This issue is similar to #8, #22, and #18, which have all been closed. I am using v1.2.0.0 of the Stream Deck app. I have a laptop with 3 external monitors, all of which have 1920x1080 native resolution. On my external monitors I am able to switch to 1600x900 and 1280x720 resolution without any issues, but cannot switch back to 1920x1080; it just shows the yellow triangle error icon. I have tried `Default`, `Stretch`, and `Center` scaling modes, but they all have the same problem.

I am using Windows 10, and this is how my displays are configured, where 1 is the laptop monitor and 2, 3, and 4 are external monitors.

![image](https://github.com/user-attachments/assets/c08fc36f-bc9d-4464-b6b5-9eaf60ae2c7a)

I thought perhaps having displays vertically stacked was causing the problem, so I changed to this and retried, but still had the same issue.

![image](https://github.com/user-attachments/assets/505d1e5d-72a7-4a84-84fb-ebf74759845f)

[This comment](https://github.com/GeekyEggo/windows-display-settings/issues/8#issuecomment-1731613598) in issue #8 said to install the [Custom Resolution Utility](https://customresolutionutility.net/) and run the `reset-all.exe`, then reboot. I did that, and it reset all of my displays so I had to set them all back to Extended Display mode and reposition them. Once I did that I was able to switch away from 1920x1080 to another resolution and then back to 1920x1080 and it worked, but only once. After I switched away from 1920x1080 again and tried to switch back, it showed the error triangle again. It did this for each of my external monitors.

I'll note though that this issue only happens with my external monitors. I am able to change my main laptop display resolution away from and back to 1920x1080 without any issues. I thought perhaps it was because my laptop screen is the "Main display", so I set an external monitor as the main display and tried again. Once again I was able to have it change the external monitor back to 1920x1080 one time, but all subsequent attempts fail.

So because the external monitors are able to switch to 1920x1080 one time after having their display configuration reset, it leads me to believe that it is not a hardware issue (i.e. something specific to the monitor or the cable). It seems that switching to 1920x1080 sets something in the configuration, that once set, does not allow switching to 1920x1080 again later.

**Comment by deadlydog** [comment]
I forked the repo to see if I could debug the problem myself, since I can reproduce it on my machine obviously. I can't figure out how to debug the Stream Desk plugin though. When I run the app it launches Stream Deck, and I attached to the process, but it shows that no symbols are loaded so it doesn't hit any breakpoints. All of the info I find online seems to be for using the JavaScript SDK, not a C# one. Are you able to point me to some docs on how to debug the plugin from Visual Studio? It would be great to add that info directly in this repo.

**Comment by AlexGurev** [comment]
> I forked the repo to see if I could debug the problem myself, since I can reproduce it on my machine obviously. I can't figure out how to debug the Stream Desk plugin though. When I run the app it launches Stream Deck, and I attached to the process, but it shows that no symbols are loaded so it doesn't hit any breakpoints. All of the info I find online seems to be for using the JavaScript SDK, not a C# one. Are you able to point me to some docs on how to debug the plugin from Visual Studio? It would be great to add that info directly in this repo.

Did you manage to solve the problem? I have the same.

**Comment by deadlydog** [comment]
No, I was waiting to get some guidance on how to debug this C# Stream Deck app before looking into it again.

**Comment by rogi** [comment]
I’m experiencing a similar issue, but in my case, I have my laptop closed and am using only an external monitor. When I manually set the resolution on the monitor to 1920x1080, I can successfully switch back to 3440x1400 on the Stream Deck. However, switching from 3440x1400 to 1920x1080 doesn’t work properly. In fact, it automatically reverts to the laptop display, which forces me to take my laptop out of its dock and open it to regain the screen.
